apps/bfd-pipeline/bfd-pipeline-rda-grpc-apps/src/main/java/gov/cms/bfd/pipeline/rda/grpc/apps/RdaServerApp.java

```diff
@@ -58,14 +58,28 @@ public static void main(String[] args) throws Exception {
     LOGGER.info("server stopping.");
   }
 
+  /** The config class to store config options for port, seed, and the number to send to s3. */
   private static class Config {
+    /** The port to use. */
     private final int port;
+    /** The seed value. */
     private final long seed;
+    /** The max number to send. */
     private final int maxToSend;
+
+    /** Sets the fiss claim file. */
     @Nullable private final File fissClaimFile;
+    /** Sets the mcs claim file. */
     @Nullable private final File mcsClaimFile;
+    /** Sets the s3 file client. */
     @Nullable private final S3JsonMessageSources s3Sources;
 
+    /**
+     * Configs the S3bucket for connectivity for Fiss and Mcs claims.
```

Comment:
Maybe this ought to be a _little_ less casual? I'm not sure ;) 
```suggestion
     * Configures the S3bucket for connectivity for Fiss and Mcs claims.
```